.drone.yml

```diff
@@ -1,84 +1,136 @@
-pipeline:
+---
+kind: pipeline
+name: default
+type: kubernetes
 
-  build_8:
-    image: node:8
-    commands:
-      - npm --loglevel warn install
-      - npm run test
-    when:
-      event: [push, pull_request]
+steps:
+- name: build
+  pull: if-not-exists
+  image: node:8
+  commands:
+  - npm --loglevel warn install
+  - npm run test
+  when:
+    event: [ pull_request, push ]
```

Comment:
lets not do buidls on pull request

Reply:
remove the pull_requests on the build steps